docker/README.md

```diff
@@ -0,0 +1,32 @@
+# RackHD on Docker instructions
+
+Copyright 2016, EMC, Inc.
+
+Install Docker Toolbox on your system. See https://www.docker.com/products/docker-toolbox
+
+This has only been tested on Mac OS X.
```

Comment:
Ok, I'll wait to merge until that @rolandpoulter, but your other stuff is merged.